src/server/mirserver-internal.pc.in

```diff
@@ -0,0 +1,8 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+includedir=@PKGCONFIG_INCLUDEDIR@
+
+Name: mirserver-internal
+Description: Mir server internal headers
+Version: @MIR_VERSION@
+Requires: mirserver mircommon mirplatform mircore
+Cflags: -I${includedir}/mirserver-internal
```

Comment:
I just looked at our other .pc.in files, most follow this pattern:
```suggestion
includedir=@PKGCONFIG_INCLUDEDIR@/mirserver-internal

Name: mirserver-internal
Description: Mir server internal headers
Version: @MIR_VERSION@
Requires: mirserver mircommon mirplatform mircore
Cflags: -I${includedir}
```
Obviously, it works both ways

Reply:
Agreed. I was following a non-conformative `.pc.in` file ;)